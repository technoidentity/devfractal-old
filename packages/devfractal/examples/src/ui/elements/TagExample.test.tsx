import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Tag } from './TagExample'

test('Tags Example', () => {
  const component: ReactTestRenderer = renderer.create(<Tag />)
  expect(component.toJSON()).toMatchSnapshot()
})
