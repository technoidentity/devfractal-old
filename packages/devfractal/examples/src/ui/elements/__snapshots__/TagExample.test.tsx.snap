// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Default Tag
    </h1>
    <span
      className="tag"
    >
      Tag label
    </span>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Colors
    </h1>
    <div
      className="box"
    >
      <div
        className="tags"
      >
        <span
          className="tag is-black"
        >
          Black
        </span>
        <span
          className="tag is-dark"
        >
          Dark
        </span>
        <span
          className="tag is-light"
        >
          Light
        </span>
        <span
          className="tag is-primary"
        >
          Primary
        </span>
        <span
          className="tag is-info"
        >
          Info
        </span>
        <span
          className="tag is-success"
        >
          Success
        </span>
        <span
          className="tag is-warning"
        >
          Warning
        </span>
        <span
          className="tag is-danger"
        >
          Danger
        </span>
        <span
          className="tag is-white"
        >
          White
        </span>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Sizes
    </h1>
    <div
      className="box"
    >
      <div
        className="tags"
      >
        <span
          className="tag is-normal"
        >
          Normal
        </span>
        <span
          className="tag is-medium"
        >
          Medium
        </span>
        <span
          className="tag is-large"
        >
          Large
        </span>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Modifiers
    </h1>
    <div
      className="box"
    >
      <div
        className="tags"
      >
        <span
          className="tag is-rounded"
        >
          Rounded
        </span>
        <span
          className="tag is-link"
        >
          Link
        </span>
        <span
          className="tag is-delete"
        />
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Combinations
    </h1>
    <div
      className="box"
    >
      <div
        className="tags"
      >
        <span
          className="tag is-success"
        >
          Bar
          <a
            className="delete is-small"
          />
        </span>
        <span
          className="tag is-medium is-warning"
        >
          Hello
          <a
            className="delete is-small"
          />
        </span>
        <span
          className="tag is-large is-danger"
        >
          World
          <a
            className="delete is-large"
          />
        </span>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Alignment
    </h1>
    <div
      className="box"
    >
      <div
        className=""
      >
        Centered 
      </div>
      <div
        className="tags is-centered"
      >
        <span
          className="tag is-normal"
        >
          Normal
        </span>
        <span
          className="tag is-medium"
        >
          Medium
        </span>
        <span
          className="tag is-large"
        >
          Large
        </span>
      </div>
      <div
        className=""
      >
         Right
      </div>
      <div
        className="tags is-right"
      >
        <span
          className="tag is-normal"
        >
          Normal
        </span>
        <span
          className="tag is-medium"
        >
          Medium
        </span>
        <span
          className="tag is-large"
        >
          Large
        </span>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      List of Tags
    </h1>
    <div
      className="box"
    >
      <div
        className="tags"
      >
        <span
          className="tag"
        >
          One
        </span>
        <span
          className="tag"
        >
          Two
        </span>
        <span
          className="tag"
        >
          Three
        </span>
        <span
          className="tag"
        >
          Four
        </span>
        <span
          className="tag"
        >
          Five
        </span>
        <span
          className="tag"
        >
          Six
        </span>
        <span
          className="tag"
        >
          Seven
        </span>
        <span
          className="tag"
        >
          Eight
        </span>
        <span
          className="tag"
        >
          Nine
        </span>
        <span
          className="tag"
        >
          Ten
        </span>
        <span
          className="tag"
        >
          Eleven
        </span>
        <span
          className="tag"
        >
          Twelve
        </span>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Attach Tags Together
    </h1>
    <div
      className="box"
    >
      <section
        className="section"
      >
        <div
          className="tags has-addons"
        >
          <span
            className="tag"
          >
            Package
          </span>
          <span
            className="tag is-primary"
          >
            Bulma
          </span>
        </div>
        <section
          className="section"
        >
          <div
            className="field is-grouped is-grouped-multiline"
          >
            <div
              className="control"
            >
              <div
                className="tags has-addons"
              >
                <span
                  className="tag is-dark"
                >
                  npm
                </span>
                <span
                  className="tag is-success"
                >
                  0.5.0
                </span>
              </div>
            </div>
            <div
              className="control"
            >
              <div
                className="tags has-addons"
              >
                <span
                  className="tag is-dark"
                >
                  Technology
                </span>
                <span
                  className="tag is-delete"
                />
              </div>
            </div>
          </div>
        </section>
      </section>
    </div>
  </section>,
]
`;
